docs/source/developers/cpp/windows.rst

```diff
@@ -99,6 +113,52 @@ source.
    from ``conda-forge``. You can examine the installed packages in your
    environment (and their origin) with ``conda list``
 
+Using vcpkg for build dependencies
+========================================
+
+`vcpkg <https://github.com/microsoft/vcpkg>`_ is an open source package manager
+manager from Microsoft. It hosts community-contributed ports of C and C++
```

Comment:
"manager" is repeated here.

Reply:
Thanks! Fixed in 945642a8b6bc59c7b384ef17fefe2af76229de3e